custom_dc/troubleshooting.md

```diff
@@ -43,6 +43,13 @@ Failed to create metadata: failed to create secret manager client: google: could
 
 This indicates that you have not specified API keys in the environment file. Follow procedures in [One-time setup steps](/custom_dc/quickstart.html#setup) to obtain and configure API keys.
 
+### "SQL schema check failed"
+
+This error indicates that there is a problem with the database schema. Check for the following additional errors:
+
+- "The following columns are missing..." -- This indicates that there is has been an update to the schema in the latest Docker build. To remedy this, restart the data management Docker container and then try again to start the services container.
```

Comment:
Perhaps, re<i>run</i> the data management docker container?

Also, "restart the services container" vs "try again to start the services container"?

Reply:
Done.